docker.go

```diff
@@ -343,6 +365,25 @@ func (c *DockerContainer) Terminate(ctx context.Context) error {
 	c.sessionID = ""
 	c.isRunning = false
 
+	// Remove additional volumes if any.
```

Comment:
suggestion: can you add a TODO to simplify this when when we perform the client refactor please.